comps/comps-foreman-rhel7.xml

```diff
@@ -195,6 +195,7 @@
       <packagereq type="default">rubygem-kafo_wizards</packagereq>
       <packagereq type="default">rubygem-little-plugger</packagereq>
       <packagereq type="default">rubygem-logging</packagereq>
+      <packagereq type="default">rubygem-multipart-post</packagereq>
```

Comment:
This should be in comps-foreman-plugins-rhel7.xml if you intend to keep it in the plugins repo, not this file.